mmedit/datasets/pipelines/augmentation.py

```diff
@@ -145,6 +148,190 @@ def __repr__(self):
         return repr_str
 
 
+@PIPELINES.register_module()
+class RandomResizedCrop(object):
+    """Crop the given numpy array to random size and aspect ratio.
+
+    A crop of random size (default: of 0.08 to 1.0) of the original size and a
+    random aspect ratio (default: of 3/4 to 4/3) of the original aspect ratio
+    is made. This crop is finally resized to given size.
+
+    This code is partially adopted from
+    torchvision.transforms.RandomResizedCrop:
+    [https://pytorch.org/vision/stable/_modules/torchvision/transforms/\
+        transforms.html#RandomResizedCrop].
+
+    Args:
+        keys (list[str]): The images to be resized and random-cropped.
+        crop_size (int | tuple[int]): Target spatial size (h, w).
+        scale (tuple[float]): Range of size of the origin size cropped.
+                              Default: (0.08, 1.0)
+        ratio (tuple[float]): Range of aspect ratio of the origin aspect ratio
+                              cropped. Default (3./4., 4./3.)
+        interpolation (str): Algorithm used for interpolation:
+            "nearest" | "bilinear" | "bicubic" | "area" | "lanczos".
+            Default: "bilinear".
+    """
```

Comment:
Add this line to docstring:

`Modified keys are the attributes specified in "keys".`